### PR TITLE
Core team membership changes

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**

### CODE REVIEW

**Comment by nikomatsakis** [comment]
LGTM.

File: posts/inside-rust/2020-10-23-Core-team-membership.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: "Core team membership changes"
+author: Mark Rousskov
+team: The Core Team <https://www.rust-lang.org/governance/teams/core>
+---
+
+The core team has had a few membership updates in the last month, and we wanted to provide an update.
```
Comment (nikomatsakis) [inline posts/inside-rust/2020-10-23-Core-team-membership.md:8]:
```suggestion
The core team has had a few membership updates in the last month and we wanted to provide an update.
```

I can't tell if there should be a comma here or not :P whichever looks better to you

Comment (Mark-Simulacrum) [reply posts/inside-rust/2020-10-23-Core-team-membership.md:8]: Seems fine as is

Comment (Manishearth) [reply posts/inside-rust/2020-10-23-Core-team-membership.md:8]: Accepted style is using commas in compound sentences, if we care about that .

I tend to use them when I actually want it to feel pause-y

**Review by Manishearth — APPROVED** [review]

**Pull request merged**